bagua/torch_api/moe/__init__.py

```diff
@@ -0,0 +1 @@
+from .layer import MoE  # noqa: F401
```

Comment:
move the whole moe directory to `bagua/torch_api/model_parallel/moe`

Reply:
done